### ISSUE TITLE
Inconsistency in patsy requirement (README vs requirements file))

### ISSUE THREAD

**Issue by krishnakumarg1984** [alan-turing-institute/mogp-emulator#241]
In the README, it says `patsy` is strongly recommended and talks about the optional requirements text file.

However, the `requirements.txt` file implies that `patsy` is a requirement and not just a strong recommendation. 

It would be helpful to newcomers if this discrepancy can be resolved suitable, and the files made consistent.

**Comment by edaub** [comment]
Yes, this is correct. `patsy` is now a requirement and the readme need to be updated. When installing from `pip` this should automatically be handled as `pip` uses the `requirements.txt` file for installation.
